backend/private-graph/graph/schema.graphqls

```diff
@@ -2397,18 +2401,20 @@ type Query {
 		limit: Int
 	): LogConnection!
 	logs_histogram(project_id: ID!, params: QueryInput!): LogsHistogram!
+	# deprecated - use `metrics` instead
 	logs_metrics(
 		project_id: ID!
 		params: QueryInput!
-		column: String!
-		metric_types: [MetricAggregator!]!
+		column: String # deprecated - use `expressions` instead
+		metric_types: [MetricAggregator!] # deprecated - use `expressions` instead
```

Comment:
would we be able to remove these after we migrate the old stored viz configs?

Reply:
unfortunately no since older versions of the Grafana integration will still be using these deprecated graphql resolvers